.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@josemp10 Can you please delete this file? This is your Eclipse configuration and IDE specifics file shouldn't be in the repo